cluster-up/cluster/kind/podpreset.sh

```diff
@@ -20,7 +21,7 @@ function podpreset::validate_admission_plugin_is_enabled() {
     local -r wait_time=$2
     local -r control_plane_container="$cluster_name-control-plane"
 
-    if ! timeout "${wait_time}s" bash <<EOT
+    if ! timeout "${wait_time}" bash <<EOT
```

Comment:
I find this "closure pattern" a bit awkward. Is that a common pattern? Can't we express this with a normal function declaration and a call to it?

Reply:
Unfortunatly `timeout` won't work with a function name, its expects a binary to execute.
We workaround it by passing `bash <command>`
this way it will run the code we want in place.

Reply:
OK, thanks!